plugins/i18n_subsites/i18n_subsites.py

```diff
@@ -270,7 +270,13 @@ def install_templates_translations(generator):
     Only if the 'jinja2.ext.i18n' jinja2 extension is enabled
     the translations for the current DEFAULT_LANG are installed.
     '''
-    if 'jinja2.ext.i18n' in generator.settings['JINJA_EXTENSIONS']:
+    if 'JINJA_ENVIRONMENT' in generator.settings: # pelican 3.7+
+        jinja_extensions = generator.settings['JINJA_ENVIRONMENT'].get(
+            'extensions', [])
+    else:
+        jinja_extensions = generator.settings['JINJA_EXTENSIONS']
+
+    if 'jinja2.ext.i18n' in jinja_extensions:
```

Comment:
@dbrgn: you do realize this code comes from upstream pelican-plugins? Probably to make the plugin compatible with both pelican-3.7 and earlier. 

But I fully agree to also update requirements.txt to 3.7.1.